CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## [0.21.0](https://github.com/redhat-developer/quarkus-ls/milestone/24?closed=1) (December 3, 2024)
+
+### Enhancements
+
+- Data model template matcher support. See [#1021](https://github.com/redhat-developer/quarkus-ls/pull/1021).
+- Proper fix for comments in TemplateExtension quickfix test. See [#1020](https://github.com/redhat-developer/quarkus-ls/pull/1020).
+- Add completion support for quarkus.langchain4j.ollama.embedding-model.model-id. See [#1014](https://github.com/redhat-developer/quarkus-ls/issues/1014).
+- Support for multiple Qute template root paths. See [#1005](https://github.com/redhat-developer/quarkus-ls/pull/1005).

```

Comment:
I think this was released in 0.20.0

Reply:
No, multiple template root paths is a 0.21.0 feature